src/apps/Altinn.Authorization/src/Altinn.Authorization/Constants/AltinnXacmlConstants.cs

```diff
@@ -115,6 +115,11 @@ public static class MatchAttributeIdentifiers
             /// AccessPackage Attribute match identifier
             /// </summary>
             public const string AccessListAttribute = "urn:altinn:access-list";
+
+            /// <summary>
+            /// Party Id attribute match indentifier 
+            /// </summary>
+            public const string PartyUUIdAttribute = "urn:altinn:party:uuid";
```

Comment:
Finnes allerede definert i XacmlRequestAttribute

MatchAttributeIdentifiers brukes vel hovedsakelig for attributter som brukes i policy (ikke at det faktisk er noe definert/ryddig bruk av dette i dag heller)